examples/telemetry-monitoring/prometheus/prometheus.yml

```diff
@@ -0,0 +1,17 @@
+global:
+  scrape_interval:     1s
```

Comment:
Just one note. This would a little too bit aggressive for production, but for test/example is ok.